lib/tests/ruby/ruby.cc

```diff
@@ -528,7 +528,7 @@ SCENARIO("custom facts written in Ruby") {
         }
     }
     GIVEN("a custom on_message block") {
-        log_capture capture(level::debug);
+        log_capture capture(level::warning);
```

Comment:
file_readable is fine; the idea is to make it clear whether the file is missing, or unreadable because of permissions.